drivers/riello_usb.c

```diff
@@ -972,6 +979,13 @@ void upsdrv_initinfo(void)
 	dstate_addcmd("test.battery.start");
 	dstate_addcmd("test.panel.start");
 
+	dstate_setinfo("ups.delay.shutdown", "%d", offdelay);
+	dstate_setflags("ups.delay.shutdown", ST_FLAG_RW | ST_FLAG_STRING);
+	dstate_setaux("ups.delay.shutdown", 3);
+	dstate_setinfo("ups.delay.reboot", "%d", bootdelay);
```

Comment:
Here and above, the dstate assignment from `unsigned int` should not be formatted as `%d` but as `%u`, right?